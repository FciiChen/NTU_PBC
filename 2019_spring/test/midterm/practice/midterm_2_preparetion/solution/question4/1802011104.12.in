沉默著，一言不發，林中一片死寂的安靜，明明沒有人有任何動作，氣氛卻緊張得一觸即發。

　　卻有人若無其事的開口。

　　「偌大皇朝，到現在還在以命相拚的，只剩下你們血浮屠。」那孩子語氣輕輕，微帶惋惜，「我不得不說，你們真是……愚忠。」

　　「看見他的下場了嗎？」他指指地上那一團，小小年紀，面對那樣的慘景依舊氣定神閒，平靜漠然得令人心中發冷，「你再執迷不悟，也一樣。」

　　男子卻已將目光緩緩收回，看向那孩子，竟然還笑了一下。

　　「大成皇朝最起碼還有我們這群願意戰至最後一刻的愚忠……」他笑，「就不知道將來閣下家皇權崩塌之時，有幾個人會為你赴死？」

　　「很遺憾，你看不到那一天。」那孩子並不生氣，微微一笑，語氣一轉，「但是，就算你看不到，你不希望你的子孫後代，能看到那一天嗎？」

　　男子面色一變。

　　「你家族世代子嗣艱難。」那孩子看著他，語氣淡淡，「到了你這一代，百年難遇的有了兄弟兩人，但是就算如此，好運似乎也已經走到盡頭，你那兄弟雖然早早娶妻，至今卻只有一個男丁，據說還是個……」他說到這裡，輕笑一下住了口。

　　男子臉色鐵青，一直穩定的雙手，竟然微微有些發抖，他注視這小小孩子，眼神中終於有了幾分震驚。

　　血浮屠的一切都是絕密，屬於他這個首領、屬於他家族的隱私，更是世上幾乎無人得知，這個小小孩子，竟然瞭如指掌！

　　那孩子卻無視他的臉色，坦然繼續，「我相信你不懼身死，也認為金銀珠玉買不動世代忠誠的血浮屠首領，但是我相信，世代守護血浮屠第三十七代家主，一定不願意家族承繼在自己手中，徹底斷絕。」

　　輕輕巧巧一句話，卻如巨錘般砸中男子，他踉蹌退後一步，臉色慘然。

　　世上沒有怕死的英雄，卻有被責任所困的蛟龍。

　　家族一脈今日絕，他至死難見先祖。

　　那孩子看著他神色，嘴角彎起一抹滿意的弧度：「我不傷你，我甚至不問你任何事情，只要你此刻放下這包裹，轉身而去，你家族的那個孩子，從此便會安枕無憂。」

　　豎起手掌，尚帶童稚的聲音聽來竟也錚錚有聲：「以我聖寧血脈為誓，違者，斷嗣！」

　　林中眾人齊齊動容——一手掀翻大成皇朝統治的寧氏家族，是大成皇朝外戚之族，據說百年前是大成屬國皇室血脈分支，百年前被大成吞併，因此寧傢俬下自號為聖，極重血統承繼，這樣的誓言，是相當重了。

　　男子表情不變，眼神中卻已露出沉吟之色，顯見已被他的誓言打動。

　　「拿來吧……」那孩子察言觀色，立即輕輕伸出雙手，舒展向前，一個等待接過的姿勢。

　　密林黝黯的色彩裡，腕骨精緻掌心如玉，語聲如一縷細絲悠悠散開，纏纏繞繞捆上男子驛動不安的心神。

　　「血浮屠只剩下你一個……普天之下，只要這裡的人不說，誰也不會知道你曾做過什麼……」低沉的聲音聽來無盡誘惑，幽幽蠱惑人心，「你只要放開手，從此之後，天下再無人可以為難你家族……」

　　男子沉默著，似在思量，眼神悲涼而遙遠，似乎想透過此刻暗沉的天幕，看見想要看見的人。

　　眾人屏息凝神等著，等著他退，或進。

　　等著自己成為這個輝煌皇朝的終結者，等著這皇朝最後一點星火熄滅。

　　這一刻沉默厚重宛如實質，泥漿般凝結，將眾人身心動作都似要束縛。

　　很久以後。

　　男子終於抬頭，望定他，露出一個笑容。

　　那笑意輕淺，深重晦暗的色彩裡，看來浮薄如早間的霧氣。

　　那孩子眯著眼睛，眼神裡掠過一絲寒芒。

　　男子的手，卻已經抬了起來，掌心微赤，顯見已經提足了真力。

　　那孩子眼神收縮得更緊，身形卻紋絲不動。

　　那男子提掌，卻不是放開的姿勢，而是突然向下一沉。

　　沉向懷中錦緞包裹的前心！

　　與此同時悲憤的笑聲激越蕩起，震得這林中落葉簌簌而下。

　　「國將傾亡，何來家族？既然如此，不如都毀個乾淨！」

　　眉頭一動，那孩子剎那間輕煙般掠了過來，與此同時密林四週一直虎視眈眈的身影都動了，灰色暗影如收束的網，四面收攏，勢必要將男子手中的動作阻止。

　　然而他們動作再快，又如何能比落掌的速度，隱約間紅光一現，手掌已經按上包裹。

　　「嗚——」

　　半聲嗚咽尚未響起，便已戛然斷絕！

　　那聲音那般細弱稚嫩，在午夜風雨密林中，如殘燭星火，剎那飄搖，轉瞬消逝。

　　所有人面色鐵青。

　　少年的眼神，一層層的冷了下來，他盯著男子，明明身形尚小氣勢未足，看來卻如一條幼龍於長天之上盯住了山野大地上奔馳的虎。

　　只是那眼神在掠過那已經毫無動靜的包裹時，依然有幾分狐疑。

　　那男子卻隨手將包裹一拋，憤聲笑道：「既已與皇朝同殉，也無所謂葬在哪裡！」

　　包裹飛了出去。

　　眾人齊齊仰頭，看著飛龍舞鳳的錦緞包裹在半空中劃過一條金色的弧線，以一種驚心動魄的弧度迅速落向密林後的崖下。

　　少年眉一揚，飛快叱道：「攔下！」

　　立時有人騰身而起，男子卻飛身掠了過來，直直撲向少年，半空裡手一掣寒光閃耀，罡風呼嘯劈向少年天靈蓋。

　　所有人驚呼出聲趕緊回轉，再也無心去追那個包裹，男子卻在將要撲向少年身前時突然長聲一笑：「血浮屠與皇朝共存亡，不敢多活一刻！」

　　他手一抄撈起地上那團看不清臉面的血肉，身形一扭，比那包裹更快的衝向崖下。

　　眾人不想他在萬里奔逃筋疲力盡時刻依然有如此速度，一時都追不及，眼看他放棄對主子的攻擊，半空中都舒了一口長氣。

　　不想驚變突起！

　　「轟！」

　　天地灰濛中突然迸開明烈的色彩，半空中騰起一朵烏金色的花，巨大的氣浪將穿林而入的綿綿雨絲激飛，下了一道斑斕瘆人的慘烈血肉之雨。

　　一片深黑亮紅腥霧瀰漫裡，正當其衝的那金尊玉貴的孩子，無聲無息的倒了下去。

　　四面驚呼聲都似要凝結！

　　良久，一些淡紅的碎肉，撲簌簌自樹葉之端無聲滑落，瞬間在人腳下積了一堆，那是剛才被扔在少年腳前的血浮屠衛士最後遺骸。

　　剛才男子看似拎起屍體離開，卻在敵人最不防備的那一刻，引爆了藏在屍體中的炸藥。

　　衣袂帶風聲瑟瑟，所有人都向倒地生死不知的主子趕了過去。

　　卻有一聲愴然長笑，自未散硝煙之中響起。
BREAK