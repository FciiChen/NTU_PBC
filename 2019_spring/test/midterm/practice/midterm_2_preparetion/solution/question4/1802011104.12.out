那_子 12
血_屠 7
了_來 4
男_卻 4
下_了 3
了_去 3
你_族 3
半_中 3
向_年 3
大_皇 3
